src/viam/sdk/components/private/pose_tracker_server.cpp

```diff
@@ -0,0 +1,63 @@
+#include <viam/sdk/components/private/pose_tracker_server.hpp>
+
+#include <viam/api/component/posetracker/v1/pose_tracker.pb.h>
+
+#include <viam/sdk/common/service_helper.hpp>
+#include <viam/sdk/common/utils.hpp>
+#include <viam/sdk/components/pose_tracker.hpp>
+#include <viam/sdk/config/resource.hpp>
+#include <viam/sdk/rpc/server.hpp>
+
+namespace viam {
+namespace sdk {
+namespace impl {
+
+PoseTrackerServer::PoseTrackerServer(std::shared_ptr<ResourceManager> manager)
+    : ResourceServer(std::move(manager)) {}
+
+::grpc::Status PoseTrackerServer::GetPoses(
+    ::grpc::ServerContext*,
+    const ::viam::component::posetracker::v1::GetPosesRequest* request,
+    ::viam::component::posetracker::v1::GetPosesResponse* response) noexcept {
+    return make_service_helper<PoseTracker>(
+        "PoseTrackerServer::GetPoses", this, request)([&](auto& helper, auto& pose_tracker) {
+        const std::vector<std::string> body_names(
```

Comment:
It doesn't look like we're actually using `body_names` anywhere, I think we can get rid of this.

Reply:
oops yes good catch!